### PR TITLE
Replacing eval with get_nms_from_type function

### PR DESCRIPTION
**Pull request by SemyonBevzuk**
Added function `get_nms_from_type`  to define nms function by nms type.

### CODE REVIEW

**Comment by druzhkov-paul** [comment]
Please elaborate what is the purpose of this?

**Comment by SemyonBevzuk** [comment]
Leonid asked to replace function `eval` with something else.

File: mmdet/ops/nms/nms_wrapper.py [hunk 1]
```diff
@@ -154,7 +169,7 @@ def batched_nms(boxes, scores, idxs, nms_cfg, class_agnostic=False):
             boxes_for_nms = boxes + offsets[:, None]
 
     nms_type = nms_cfg_.pop('type', 'nms')
-    nms_op = eval(nms_type)
+    nms_op = get_nms_from_type(nms_type)
```
Comment (druzhkov-paul) [inline mmdet/ops/nms/nms_wrapper.py:172]: What about `ast.literal_eval`?

**Review by druzhkov-paul — COMMENTED** [review]
Please place your branch directly in this repo.

File: mmdet/ops/nms/nms_wrapper.py (hunk 1 above)
Comment (SemyonBevzuk) [reply mmdet/ops/nms/nms_wrapper.py:172]: As I understand, `ast.literal_eval` will not work in this situation. Here is the information from the [documentation](https://docs.python.org/3/library/ast.html#ast.literal_eval):
> The string or node provided may only consist of the following Python literal structures: strings, bytes, numbers, tuples, lists, dicts, sets, booleans, None and Ellipsis.

So, `ast.literal_eval` cannot process `nms` function.

**Comment by SemyonBevzuk** [comment]
> Please place your branch directly in this repo.

Done, PR #272

**Pull request closed without merging**